package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "formFieldAppender",
-    "version": "1.0.4",
+    "version": "1.0.5",
```

Comment:
This would probably be a minor version bump rather than a bugfix bump, since you added new (non-breaking) functionality